src/TwitterFeed.php

```diff
@@ -77,6 +78,7 @@ public function __construct(
         $this->userId = $userId;
         $this->excludeReplies = $excludeReplies;
         $this->includeRts = $includeRts;
+        $this->extended = $extended;
```

Comment:
@pcgc Be careful, `$extended` var is not defined in your constructor. You can define as constructor optional argument.

Reply:
line 67 
`$extended = false`

Reply:
My bad